docs/rest_api.rst

```diff
@@ -367,6 +368,10 @@ The ``balance`` field will signify the initial deposit you wish to make to the c
 The request to the endpoint should later return the fully created channel object
 from which we can find the address of the channel.
 
+**OBS:** DO NOT set ``reveal_timeout`` if you don't know what you're doing. This option
```

Comment:
Is `**OBS:**` some special `.rst` tag? Otherwise I'd suggest using something more obvious, such as `**Caution:**`

Reply:
Done.